docs/documentation/platform/identities/kubernetes-auth.mdx

```diff
@@ -0,0 +1,233 @@
+---
+title: Kubernetes Auth
+description: "Learn how to authenticate with Infisical in Kubernetes"
+---
+
+**Kubernetes Auth** is a Kubernetes-native authentication method for applications (e.g. pods) to access Infisical.
+
+## Diagram
+
+    The following sequence digram illustrates the Kubernetes Auth workflow for authenticating applications running in pods with Infisical.
+
+```mermaid
```

Comment:
Does this render for you?
<img width="885" alt="Screenshot 2024-05-17 at 9 30 27 PM" src="https://github.com/Infisical/infisical/assets/9300960/5ca2e9e5-3a43-4b32-868e-0abdad53944a">

Reply:
Yup it rendered for me.